cmake/std/atdm/utils/atdm_config_helper_funcs.sh

```diff
@@ -65,3 +65,14 @@ function get_sparc_dev_module_name() {
   #echo "sparc_module_name = '${sparc_module_name}'"
   echo "${sparc_module_name}"
 }
+
+#
+# Remove the "substring" from the PATH var.
+# @param substring: the string to remove.
+#
+function atdm_remove_from_path() {
+  local substring=$(printf "%s" "$1" | sed 's/\//\\\//g')
+  local path=$(printf "%s" "$PATH" | sed "s/$substring://g")
+  path=$(printf "%s" "$PATH" | sed "s/:$substring//g")
+  export PATH=$path
+}
```

Comment:
It might be more straightforward and safe to split this on ':' into a bash array, then loop over removing entries that match the input list, and putting it back together again?

Reply:
I wrote some very similar bash code in:

* https://github.com/trilinos/Trilinos/blob/master/commonTools/framework/get-changed-trilinos-packages-helpers.sh

I think if you replace 'comma' with 'colon' in these names and `,` with `:`, I think you have the same algorithm.  And thinking of this more, we could have:

```
atdm_remove_paths_from_path_dir PATH <path1> <path2> ...
```

And we can write some unit tests for that using the shunit2 framework.